packages/js/product-editor/src/blocks/track-inventory/editor.scss

```diff
@@ -0,0 +1,15 @@
+.wp-block-woocommerce-product-track-inventory-fields {
```

Comment:
Is there a reason we're naming some of these `editor.scss` and others `style.scss`, or should all of them be `editor.scss` now?